config/initializers/exercism_config.rb

```diff
@@ -12,3 +12,10 @@
 Exercism.config.test_runner_orchestrator_url = services_config[:test_runner_orchestrator_url]
 Exercism.config.analyzer_orchestrator_url = services_config[:analyzer_orchestrator_url]
 Exercism.config.representer_orchestrator_url = services_config[:representer_orchestrator_url]
+
+Exercism.config.dynamodb_client =
+  Aws::DynamoDB::Client.new(
+    Exercism.config.aws_auth.merge({
+                                     endpoint: "http://localhost:3039"
```

Comment:
We should configure the hostname of this endpoint, as a different host name will be used when using docker-compose

Reply:
This has now moved into the Exercism.config, which will be set via the bin script.